### PR TITLE
add&resprite furnitures

### PR DESCRIPTION
**Pull request by Jaraten**
resprite and add furnitures and wallmount structures, tiles and other.

### CODE REVIEW

File: Resources/Prototypes/_CP14/Entities/Structures/Furniture/beds.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 75
+      behaviors:
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          MaterialWoodPlank:
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Structures/Furniture/beds.yml:45]: we should have our own kind of wood, not the standard kind.

File: Resources/Prototypes/_CP14/Entities/Structures/Furniture/dressers.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 50
+
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          MaterialWoodPlank1:
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Structures/Furniture/dressers.yml:39]: we should have our own kind of wood, not the standard kind.

File: Resources/Prototypes/_CP14/Entities/Structures/Furniture/wallmount.yml [hunk 3]
```diff
@@ -105,4 +106,50 @@
...
-            acts: [ "Destruction" ]
\ No newline at end of file
+            acts: [ "Destruction" ]
+
+- type: entity
+  id: CP14WallmountBarShelfA
+  name: bar shelf
+  suffix: 1
+  parent: BaseWallmount
+  description: buba. ##TODO
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Structures/Furniture/wallmount.yml:116]: buba!

File: Resources/Textures/_CP14/Objects/Materials/wood.rsi/log.png
Comment (TheShuEd) [inline Resources/Textures/_CP14/Objects/Materials/wood.rsi/log.png:1]: need edit meta.json copyright, to reflect authorship

File: Resources/Textures/_CP14/Objects/Materials/wood.rsi/planks_3.png
Comment (TheShuEd) [inline Resources/Textures/_CP14/Objects/Materials/wood.rsi/planks_3.png:1]: need edit meta.json copyright, to reflect authorship

File: Resources/Textures/_CP14/Structures/Furniture/Tables/wood.rsi/state0.png
Comment (TheShuEd) [inline Resources/Textures/_CP14/Structures/Furniture/Tables/wood.rsi/state0.png:1]: need edit meta.json copyright, to reflect authorship

File: Resources/Textures/_CP14/Structures/Furniture/chairs.rsi/wooden.png
Comment (TheShuEd) [inline Resources/Textures/_CP14/Structures/Furniture/chairs.rsi/wooden.png:1]: need edit meta.json copyright, to reflect authorship

File: Resources/Textures/_CP14/Structures/Furniture/wood_cupboard.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
```
Comment (TheShuEd) [inline Resources/Textures/_CP14/Structures/Furniture/wood_cupboard.rsi/meta.json:3]: Maybe we should use a new project license for all new sprites, if we don't want other people to use them without our permission. But that's up to you.

**Review by TheShuEd — CHANGES_REQUESTED** [review]
First light review. I'll run more later and check it all out in the game

File: Resources/Textures/_CP14/Structures/Furniture/wood_cupboard.rsi/meta.json (hunk 4 above)
Comment (Jaraten) [reply Resources/Textures/_CP14/Structures/Furniture/wood_cupboard.rsi/meta.json:3]: Oh, yeah, that totally slipped my mind. I'll change it.


> Maybe we should use a new project license for all new sprites, if we don't want other people to use them without our permission. But that's up to you.

Oh, yeah, that totally slipped my mind. I'll change it.

**Review by TheShuEd — APPROVED** [review]

**Pull request merged**